airflow/config_templates/default_airflow.cfg

```diff
@@ -434,6 +434,10 @@ force_log_out_after = 0
 # The UI cookie lifetime in days
 session_lifetime_days = 30
 
+# Have the webserver save the dagbag to the DB every refresh. Set to true if you're
```

Comment:
Refresh of what? A web page, i.e, on any GET request? Or when the gunicorn master restarts?